jsweb/exercises/dom_detective.html

```diff
@@ -1,68 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <title>Project: DOM Detective</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="">
-    <meta name="author" content="">
-    <link rel="stylesheet" type="text/css" href="../../common/bootstrap-responsive.css">
-    <link rel="stylesheet" type="text/css" href="../../common/bootstrap.css">
-  </head>
- <body>
-<div class="container">
-<h2>Project: DOM Detective</h2>
-
-
-<h3 class="page-header">The case of the GirlDevelopIt website</h3>
-
-<ul>
- <li>Open up <a href="http://www.girldevelopit.com">girldevelopit.com</a> in Chrome or Firefox,
- and open up the console (CMD-Option-I on Macs).
- <li>Use <a href="https://www.khanacademy.org/computing/computer-programming/html-css-js/html-js-dom-access/a/summary-dom-access-methods">DOM access methods</a> to find the following parts of the page. You probably first want to use Inspect Element to figure out what their tag names, classes, and IDs are.</li>
-      <ol>
-     <li>Every image on the page</li>
-     <li>The navigation area in the upper right</li>
-     <li>The MailChimp sign-up form in the bottom</li>
-     <li>The upper left-hand logo that says GDI (Hint: use CSS descendant selectors)</li>
-     <li>The logos of the media sources that featured GirlDevelopIt (lifeHacker, TED, etc., at the bottom of the page)</li>
-     <li>The big heading that says "DON'T BE SHY DEVELOP IT"</li>
-     <li>BONUS: all the dots in the map</li>
-   </ol>
- </ul>
- <br><br>
-<pre id="solution1" style="display:none;">
-document.getElementsByTagName("img");
-document.getElementsByClassName("nav");
-document.getElementById("mc_embed_signup");
-document.querySelector(".logo img");
-document.querySelectorAll(".press-logos img");
-document.querySelector(".opener h1");
-document.querySelectorAll("circle");
-</pre>
-
-  <h3 class="page-header">The case of the BrainPickings blog</h3>
-
-<ul>
- <li>Open up <a href="http://www.brainpickings.org">brainpickings.com</a> in Chrome or Firefox,
- and open up the console.
- <li>Use <a href="https://www.khanacademy.org/computing/computer-programming/html-css-js/html-js-dom-access/a/summary-dom-access-methods">DOM access methods</a> to find the following parts of the page. You probably first want to use Inspect Element to figure out what their tag names, classes, and IDs are.</li>
-    <ol>
-     <li>The bylines of each blog post (the text that displays the author)</li>
-     <li>Every image that is inside a blog post</li>
-     <li>Every link inside the left-hand nav bar</li>
-     <li>The Facebook link in the top right social media bar</li>
-     <li>The input box for entering your name for email subscription, on the left side</li>
-     <li>Every link inside the tag cloud on the left side</li>
-   </ol>
- </ul>
-
-  <script>
-function showSolution(num) {
```

Comment:
I followed your instructions for git mv exactly, and this is what resulted. 

Per my earlier note, I think I have to git reset all changes to those files, run git mv, and force push up my changes from scratch for me to do what you're asking me to do. I just changed the grammar/ wording of the exercises (not the actual content). If you'd like, you can open the JS web DOM slides locally and see what the exercise slides point to.